src/main/java/com/sparkplatform/api/models/DateField.java

```diff
@@ -22,6 +22,9 @@ public DateField(String str) {
 			}
 		}
 	}
+	public DateField() {
+		value = null;
+	}
```

Comment:
This seems kind of strange, given I would expect a null instance of this class instead of an instance containing a null value. Perhaps something changed with the json bindings that made this a requirement? In any case, I think it's an OK substitute.

The only real risk is the serialization will still fail, but lower down the processing chain.